descriptors/ModuleDescriptor-template.json

```diff
@@ -101,8 +103,10 @@
             "orders-storage.reporting-codes.item.put",
             "orders-storage.reporting-codes.item.delete",
             "configuration.entries.collection.get",
-            "finance-storage.encumbrances.collection.get",
-            "finance-storage.encumbrances.item.post",
+            "finance.funds.collection.get",
```

Comment:
the POST is missing the funds permission and PUT is missing the transaction permission

Reply:
I don't think the transactions permission is needed. so just edit the POST permission